chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/RoomSpek.kt

```diff
@@ -330,6 +330,17 @@ object RoomSpek : Spek({
             assertThat(room.pushNotificationTitleOverride).isEqualTo(pnTitleOverride)
         }
 
+        it("creates room with a membership") {
+            setUpInstanceWith(createDefaultRole(), newUsers(PUSHERINO, ALICE))
+
+            val pusherino = chatFor(PUSHERINO).connect().assumeSuccess()
+
+            val room = pusherino.createRoom(name =  GENERAL, userIds = listOf(ALICE)).assumeSuccess()
+
+            assertThat(room.memberUserIds).isEqualTo(setOf(ALICE, PUSHERINO))
+            assertThat(pusherino.rooms[0].memberUserIds).isEqualTo(setOf(ALICE, PUSHERINO))
```

Comment:
As discussed, for completeness, it would be good to check the returned value, the state and the event as all those areas were touched within the ME effort.

Reply:
That makes sense - have added those extra events in for the two tests I created. If we've got time at the end of this project, I'm happy to go back and look at what other tests we could add checking the event/returned state/current user to too.